test-lit/unit/version_test.f90

```diff
@@ -0,0 +1,4 @@
+! Check that lit configuration works by checking the compiler version
+
+! RUN: f18 -V 2>&1 | FileCheck  -check-prefix=VERSION %s
+! VERSION-DAG: f18 compiler
```

Comment:
should this be a DAG check? Isn't DAG used when there are more than one check and the ordering is not significant?

Reply:
Yes, it is wrong

Reply:
Done!